app/src/main/java/com/example/neighborly/Item.java

```diff
@@ -1,5 +1,52 @@
 package com.example.neighborly;
 
-public class Item{
+import android.graphics.Bitmap;
 
+import com.google.firebase.auth.FirebaseUser;
+
+public class Item {
+
+    private Bitmap image;
```

Comment:
I saw people online using the Firebase storage system instead of saving the bitmap of the images in the "regular" Firebase DB. 
Like in here:
https://stackoverflow.com/questions/50816557/storing-and-displaying-image-using-glide-firebase-android

What are your thoughts on that?

It could complicate the